bridge/src/mock.rs

```diff
@@ -330,6 +349,90 @@ impl ConcrateSygmaAsset {
 	}
 }
 
+pub struct SygmaDecimalConverter<DecimalPairs>(PhantomData<DecimalPairs>);
+impl<DecimalPairs: Get<Vec<(XcmAssetId, u8)>>> DecimalConverter
+	for SygmaDecimalConverter<DecimalPairs>
+{
+	fn convert_to(asset: &MultiAsset) -> Option<u128> {
+		match (&asset.fun, &asset.id) {
+			(Fungible(amount), _) => {
+				for (asset_id, decimal) in DecimalPairs::get().iter() {
+					if *asset_id == asset.id {
+						return if *decimal == 18 {
+							Some(*amount)
+						} else {
+							type U112F16 = FixedU128<U16>;
```

Comment:
One question, does `FixedU128<U16>` act the same as `fixed::types::U112F16`?

Reply:
Checked the code, and they are the same, see [here](https://gitlab.com/tspiteri/fixed/-/blob/master/src/types/mod.rs#L860)